e2e/test/scenarios/embedding-sdk/create-question.cy.spec.ts

```diff
@@ -52,7 +50,7 @@ describeSDK("scenarios > embedding-sdk > create-question", () => {
       cy.findByRole("button", { name: "Show editor" }).click();
 
       // Should be able to visualize the question again
-      cy.findByRole("button", { name: "Visualize" }).click();
+      cy.findByLabelText("Switch to visualization").click();
```

Comment:
1. New > Question > Orders > Count > Breakout by Created At: Month > Breakout by Product Category
2. Save
3. Visualize
4. Go to "Editor"
5. Hit back button in your browser

:x: "Editor" is selected, but "Visualization" is shown

----

1. New > Question > Orders > Count > Breakout by Created At: Month > Breakout by Product Category
2. Save
3. Visualize
4. Hit back button in your browser

:x: "Visualization" is selected, but "Editor" is shown

----

I had one more case, but it's basically the same problem.

Reply:
both cases should be fixed